qutebrowser/browser/webengine/webenginesettings.py

```diff
@@ -197,6 +197,31 @@ def _update_stylesheet():
             tab.run_js_async(code)
 
 
+def _init_focustools(profile):
+    """Initialize focus tools."""
+    old_script = profile.scripts().findScript('_qute_focustools')
```

Comment:
Since you use `profile.scripts()` three times here, you might want to do `scripts = profile.scripts()` maybe?